### PR TITLE
Revert "Update game"

### PR DESCRIPTION
**Pull request by SR123456789-maker**
nice

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (SR123456789-maker) [inline index.html:49]: very

**Review by SR123456789-maker — COMMENTED** [review]
nice

**Comment by SR123456789-maker** [comment]
nice

File: index.html (hunk 1 above)
Comment (SR123456789-maker) [inline index.html:49]: nice

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over</h2>
 
 <script>
```
Comment (saada1246-makers) [inline index.html:52]: hdg

**Review by saada1246-makers — APPROVED** [review]
nice

**Review by saada1246-makers — COMMENTED** [review]
nice

**Review by saada1246-makers — APPROVED** [review]
g1jgdndiej

**Review by SR123456789-maker — COMMENTED** [review]
nice

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over</h2>
```
Comment (SR123456789-maker) [inline index.html:51]: nice

**Review by SR123456789-maker — COMMENTED** [review]
nice

File: index.html (hunk 3 above)
Comment (SR123456789-maker) [reply index.html:51]: nice

**Pull request merged**